lib/ex_bitmex/ws.ex

```diff
@@ -0,0 +1,174 @@
+defmodule ExBitmex.Ws do
+  @moduledoc """
+  BitMEX WebSocket client.
+  """
+
+  import Process, only: [send_after: 3]
+
+  defmacro __using__(_opts) do
+    quote do
+      use WebSockex
+      require Logger
+
+      @test_mode Application.get_env(:bitmex, :test_mode)
+      @base "wss://" <> ((@test_mode && "testnet") || "www") <> ".bitmex.com/realtime"
```

Comment:
Can we bring this into a local var? It's better if we set it run-time and not compile-time: less erroneous when changing the config. Probably change `@test_mode` as well.

Reply:
In this case run-time should be fine. Will update it. Bring the constant at the top more like for the readability purpose.